binCompatTest/src/test/scala/cats/data/CatsDataMimaException.scala

```diff
@@ -0,0 +1,8 @@
+package cats.data
+import cats.implicits._
+
+object CatsDataPackagePrivateMimaExceptions {
```

Comment:
My bad. I should've checked that mima no longer falsely claims that this type of change breaks BC. So this is no longer needed? Would you mine removing it?

Reply:
done